Classes/ContextMenu/PasteReferenceItemProvider.php

```diff
@@ -72,6 +72,11 @@ protected function getAdditionalAttributes(string $itemName): array
             'reference' => 1,
         ];
 
+        // Add needed EXT:container information to reference into a container
+        if ($this->record['tx_container_parent'] > 0) {
```

Comment:
When EXT:container is not installed, then you will get an undefined array key warning. Better use

`if (($this->record['tx_container_parent'] ?? 0) > 0)`